vm/src/obj/objbyteinner.rs

```diff
@@ -311,6 +309,8 @@ impl ByteInnerSplitlinesOptions {
 
 #[allow(clippy::len_without_is_empty)]
 impl PyByteInner {
+    const ASCII_WHITESPACES: [u8; 6] = [0x20, 0x09, 0x0a, 0x0c, 0x0d, 0x0b];
```

Comment:
Can we use `is_ascii_whitespace`?

Reply:
find_byteset takes array, slice or vector instead of function. do you have any idea not to define this const?